contracts/modules/delegates/IexecOrderManagementDelegate.sol

```diff
@@ -119,4 +121,43 @@ contract IexecOrderManagementDelegate is IexecOrderManagement, DelegateBase, Sig
             emit ClosedRequestOrder(requestorderHash);
         }
     }
+
+    function computeDealVolume(
```

Comment:
Can you add tiny doc?
I would at least mention something like: "It predicts deal volume only if the deal is not yet created. Do not call this if deal already exists, use deal accessors instead."

Reply:
Maybe `predictFutureDealVolume` naming to be explicit?